dependent/aids-research-and-therapy.csl

```diff
@@ -1,18 +1,16 @@
 <?xml version="1.0" encoding="utf-8"?>
 <style xmlns="http://purl.org/net/xbiblio/csl" version="1.0" default-locale="en-US">
-  <!-- Generated with https://github.com/citation-style-language/utilities/tree/master/generate_dependent_styles/data/bmc -->
+  <!-- Generated with https://github.com/citation-style-language/utilities/tree/master/generate_dependent_styles/data/springer -->
   <info>
     <title>AIDS Research and Therapy</title>
-    <title-short>AIDS Res Ther</title-short>
     <id>http://www.zotero.org/styles/aids-research-and-therapy</id>
     <link href="http://www.zotero.org/styles/aids-research-and-therapy" rel="self"/>
-    <link href="http://www.zotero.org/styles/biomed-central" rel="independent-parent"/>
-    <link href="http://www.aidsrestherapy.com" rel="documentation"/>
+    <link href="http://www.zotero.org/styles/springer-basic-brackets" rel="independent-parent"/>
```

Comment:
Official answer for this journal: Vancouver is correct. I will aks again if this is somehow in general true.